### ISSUE TITLE
Feature request: overlaid reflections can be scaled in their transparency by the F^2 or Is or Ip value of that reflection

### ISSUE THREAD

**Issue by adas42** [HEXRD/hexrdgui#1749]
Basically as the title says, this will help get a visual representation of the expected intensities. Ideally we would have an option to choose between the three selections (F^2 or Is or Ip)

**Comment by psavery** [comment]
That sounds like a really nice idea! Never thought about doing that before.

By the way, in the "Reflections Table", where you select which HKLs to show, you can sort by F^2, and use that to help you only select reflections high in F^2.

**Comment by adas42** [comment]
Yes, I do frequently use the sort by F^2 but sometimes it is useful to have a visual sense of the relative intensity difference to expect. For example, superlattice reflections
